docs/reference/API/attr_to_event.md

```diff
@@ -0,0 +1,89 @@
+# List of PyScript Attributes to Events:
+
+PyScript provides a convenient syntax for mapping JavaScript events to PyScript events, making it easy to connect events to HTML tags.
+
+For example, you can use the following code to connect the click event to a button:
+
+```
+<button id="my-button" py-onClick="foo()">Click me</button>
```

Comment:
Maybeshould be `py-click` instead? I've been using this way in the tutorials and seems a bit confusing o have two different ways to handle click. What do you think?